releases/release-7.4.0.md

```diff
@@ -151,6 +151,12 @@ TiDB 版本：7.4.0
 
     更多信息，请参考[用户文档](/statistics.md#锁定统计信息)。
 
+* 引入系统变量禁用表的哈希连接 [#46695](https://github.com/pingcap/tidb/issues/46695) @[coderplay](https://github.com/coderplay)
+
+    表的哈希连接是 MySQL 8.0 引入的新特性，主要用于连接两个相对较大的表和结果集。但对于交易类负载，或者一部分在 MySQL 5.7 稳定运行的业务来说，选择到表的哈希连接可能会对性能产生风险。MySQL 通过[`优化器开关`](https://dev.mysql.com/doc/refman/8.0/en/switchable-optimizations.html#optflag_block-nested-loop)能够在全局或者会话级控制哈希连接的选择。TiDB 同样在新版本中引入系统变量 [tidb_opt_enable_hash_join](#) 对表的哈希连接进行控制。默认开启，如果客户非常确定执行计划中不需要选择表之间的哈希连接，则可以修改变量为 `NO`，降低执行计划回退的可能性，提升系统稳定性。 
+
+    更多信息，请参考[用户文档](#)。
+
```

Comment:
Please include the link address once the user doc is ready. @qw4990

Reply:
Done https://github.com/pingcap/docs-cn/pull/14954/files#r1349437172